src/extensions/default/bramble/lib/UI.js

```diff
@@ -111,11 +116,11 @@ define(function (require, exports, module) {
 
         var secondPaneWidth = BrambleStartupState.ui("secondPaneWidth");
         var firstPaneWidth = BrambleStartupState.ui("firstPaneWidth");
-                         
+
```

Comment:
as an aside, it's interesting that the linting hasn't caught this yet - is it set to disallow whitespace-at-the-end-of-lines?